validator/lib/jsonapi/validator/resource.rb

```diff
@@ -0,0 +1,123 @@
+require 'jsonapi/validator/document'
+require 'jsonapi/validator/relationship'
+
+module JSONAPI
+  module Validator
+    class Resource
```

Comment:
Sinatra-param and strong_parameters very good for rails, but the dependency on actionpack is not in line with the philosophy of this collection of gems.